hydra-android-gradle-plugin/src/main/resources/META-INF/gradle-plugins/com.pandora.hydra.properties

```diff
@@ -0,0 +1 @@
+implementation-class=com.pandora.hydra.HydraAndroidPlugin
```

Comment:
Need to rename meta property file, so core and Android plugin will have different IDs